platform/src/main/java/org/stellar/anchor/platform/component/sep/SepBeans.java

```diff
@@ -90,10 +91,9 @@ Sep38Config sep38Config() {
   public FilterRegistrationBean<Filter> sep10TokenFilter(JwtService jwtService) {
     FilterRegistrationBean<Filter> registrationBean = new FilterRegistrationBean<>();
     registrationBean.setFilter(new Sep10JwtFilter(jwtService));
-    registrationBean.addUrlPatterns("/sep6/deposit*");
     registrationBean.addUrlPatterns("/sep6/deposit/*");
-    registrationBean.addUrlPatterns("/sep6/withdraw*");
     registrationBean.addUrlPatterns("/sep6/withdraw/*");
+    registrationBean.addUrlPatterns("/sep6/withdraw-exchange/*");
     registrationBean.addUrlPatterns("/sep6/transaction");
     registrationBean.addUrlPatterns("/sep6/transactions*");
     registrationBean.addUrlPatterns("/sep6/transactions/*");
```

Comment:
Is it possible to consolidate the URL patterns such as `sep6/**`? I think the more patterns we have, the more likely we are to miss some filtering.

Reply:
Missing URL patterns are currently caught by our integration tests but I see what you mean. The difficulty is that, at least for SEP-6, GET `/info` is not an authenticated endpoint so a blanket `sep/**` does not work. I don't think we would be adding new endpoints that often after SEP-6 so I think it's okay to keep this for now.